index.js

```diff
@@ -1,5 +1,18 @@
 'use strict';
 
+const {
+  ArrayIsArray,
+  ArrayPrototypeConcat,
+  ArrayPrototypeIncludes,
```

Comment:
This prevents monkey patched prototypes on types like Array from breaking our core functionality, and is an approach used widely in the Node codebase.